apps/generer-synt-amelding-service/src/main/java/no/nav/registre/testnorge/generersyntameldingservice/provider/GenererSyntAmeldingController.java

```diff
@@ -22,6 +26,8 @@
     public List<ArbeidsforholdDTO> generateSyntheticAmeldinger(
             @RequestBody SyntAmeldingRequest request
     ) {
+        validateInput(request.getArbeidsforholdType());
```

Comment:
Hvis du endrer SyntAmeldingRequest til å bruke enum kan spring boot fikse validereingen for det. Kan du gjøre det?

Reply:
Ja, skal fikse;)

Reply:
Prøvde først Enum med kun "ordinaertArbeidsforhold" og "maritimtArbeidsforhold" (som er det som Dolly sender inn), men fikk code smell siden de ikke er på rett format for enum. Kan bytte det tilbake og bare overse codesmells, eller tenkte du på en annen måte å fikse det på?